src/frontend/calendar/index.js

```diff
@@ -24,6 +24,7 @@ function Calendar() {
 		year: currentYear,
 	};
 	const [ { month, year }, setDate ] = useState( currentMonthYear );
+	const titleMonthYear = new Date( year, month );
```

Comment:
This variable name is a bit misleading seeing that it's a JS date object that is returned. Based on how it's currently named I would expect it to be a string formatted by month & year.

Maybe something like `const calendarDate` is more suitable?

Reply:
> Based on how it's currently named I would expect it to be a string formatted by month & year.
Make sense. `calendarDate` sounds better, will change it.